projects/artstor-storage/src/lib/artstor-storage.service.ts

```diff
@@ -0,0 +1,127 @@
+import { Injectable, Inject, PLATFORM_ID } from '@angular/core'
+import { isDefined } from '@angular/compiler/src/util'
+import { isPlatformBrowser } from '@angular/common'
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ArtstorStorageService {
+  constructor(@Inject(PLATFORM_ID) private platformId: Object) {}
+
+  // Is this code being interpretted by a browser-based client?
+  private isBrowser: boolean = isPlatformBrowser(this.platformId)
+
+  // Server side object to hold generic user object
+  // Note: We don't need one for sessionStorage on the server
+  private localStorageData = this.isBrowser ? null : {
+    user: { data: { status: false, isLoggedIn: false, loggedInSessionLost: false } }
```

Comment:
Hmmm, I see what you're doing here, but I don't think we should be handling our generic user object here.